### PR TITLE
No jsonb column rule

### PR DESCRIPTION
**Pull request by paulius005**
https://www.loom.com/share/PR-Explain-Not-allowing-JSONB-columns-to-be-added-e210b2e9dfe546ffab015975a191c3a8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,26 @@ ALTER TABLE "users" ADD FOREIGN KEY ("level_id") REFERENCES "level" ("id") NOT V
...
+
+Example:
+```javascript
+// Invalid usage
+queryInterface.addColumn('Table', 'Column', { type: Sequelize.JSONB });
+queryInterface.changeColumn('Table', 'Column', { type: Sequelize.JSONB });
+queryInterface.createTable('Table', { Column: { type: Sequelize.JSONB } });
+
+// Valid usage
+queryInterface.addColumn('Table', 'Column', { type: Sequelize.STRING });
```
Comment (tuttleb) [inline README.md:100]: I think the intent here is to say "other column types are fine," but I worry the interpretation will be: I can JSON.stringify and use a text column for JSON data

File: lib/rules/no-jsonb-column/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+const reportMessage = `
+Adding new JSONB columns can lead to performance issues on the Database.
+Avoid adding new JSONB columns in the schema.
+
+If you need to store JSON data, consider other storage options or review
```
Comment (tuttleb) [inline lib/rules/no-jsonb-column/index.js:5]: do we have a doc or anything to link to for other options?

**Review by aditya4singla — APPROVED** [review]
noice

File: lib/rules/no-jsonb-column/index.js (hunk 2 above)
Comment (paulius005) [reply lib/rules/no-jsonb-column/index.js:5]: Nope! I'm just starting to work on the document, but I thought it couldn't hurt to knock out the lint rule PR now.

File: README.md (hunk 1 above)
Comment (paulius005) [reply README.md:100]: Do we do that anywhere in our codebase already?? I hope not. 

If that ends up being the case, we can move off a lint rule and onto a more rigid CI step.

Comment (paulius005) [reply README.md]: Updated README to not have `Sequelize.STRING` as the first example

**Pull request merged**